src/inference/src/os/lin/lin_system_conf.cpp

```diff
@@ -372,15 +384,19 @@ void parse_node_info_linux(const std::vector<std::string> node_info_table,
 
     for (auto& row : nodes_table) {
         for (int i = row[0]; i <= row[1]; i++) {
-            _cpu_mapping_table[i][CPU_MAP_NUMA_NODE_ID] = row[2];
+            auto& cpu_row = _cpu_mapping_table[i];
```

Comment:
Please share the node_info_table() which get error on master branch and can be handle by this change.